makepanda/makepanda.py

```diff
@@ -737,6 +737,7 @@ def parseopts(args):
         SmartPkgEnable("EIGEN",     "eigen3",    (), ("Eigen/Dense",), target_pkg = 'ALWAYS')
         SmartPkgEnable("ARTOOLKIT", "",          ("AR"), "AR/ar.h")
         SmartPkgEnable("FCOLLADA",  "",          ChooseLib(fcollada_libs, "FCOLLADA"), ("FCollada", "FCollada/FCollada.h"))
+        SmartPkgEnable("ASSIMP",    "libassimp", ("assimp"), "assimp")
```

Comment:
I think this should be "assimp", as there is an assimp.pc, not a libassimp.pc (based on a quick google for either term).  If there is no .pc file shipped at all, this should be the empty string.